pkg/apis/machine/v1beta1/machine_webhook.go

```diff
@@ -232,6 +242,156 @@ func getMachineDefaulterOperation(platformStatus *osconfigv1.PlatformStatus) mac
 	}
 }
 
+// NewValidatingWebhookConfiguration creates empty validation webhook configuration
+func NewValidatingWebhookConfiguration() *admissionregistrationv1.ValidatingWebhookConfiguration {
+	return &admissionregistrationv1.ValidatingWebhookConfiguration{
+		ObjectMeta: v1.ObjectMeta{
+			Name: DefaultWebhookConfigurationName,
+			Annotations: map[string]string{
+				"service.beta.openshift.io/inject-cabundle": "true",
+			},
+		},
+	}
+}
+
+// WithValidatingHooks returns validating webhooks list to populate configuration
+func WithValidatingHooks(namespace string) []admissionregistrationv1.ValidatingWebhook {
+	policy := admissionregistrationv1.Ignore
```

Comment:
why do we need all this extra variables? you can inline all of them with `pointer.StringPtr()`

Reply:
Makes sense, overlooked this.

Reply:
Actually, I can't, as it will create obscurity, such as `admissionregistrationv1.FailurePolicyType(pointer.StringPtr(admissionregistrationv1.Ignore))`